app/models/CalcDisplayModel.scala

```diff
@@ -21,7 +21,8 @@ import play.api.libs.json.{Json, OFormat}
 sealed trait CalcDisplayResponseModel
 case class CalcDisplayModel(calcTimestamp: String,
                             calcAmount: BigDecimal,
-                            calcDataModel: Option[CalculationDataModel]) extends CalcDisplayResponseModel {
+                            calcDataModel: Option[CalculationDataModel],
+                            crystalisedFlag: String) extends CalcDisplayResponseModel {
```

Comment:
rather than `crystalisedFlag` can we call it `calcStatus` and instead of having its type as a `String` can we use either a SealedTrait or an Enumeration for the possible values. That way we don't have to use strings everywhere and we can get some intellisence for the possible values :smile: 

As for the possible values, can we have `Crystalised` and `Estimate`